pkg/auth/requests/authenticate_test.go

```diff
@@ -207,12 +213,16 @@ func TestTokenAuthenticatorAuthenticate(t *testing.T) {
 		userLister:          userLister,
 		clusterRouter:       clusterrouter.GetClusterID,
 		refreshUser:         userRefresher.refreshUser,
+		now: func() time.Time {
+			return now
+		},
```

Comment:
Can this be simplified to `now: now,` ?

Reply:
Nope. The `now` is a specific `time.Time` value whereas `tokenAuthenticator.now` is `func() time.Time`.

Reply:
Checking back I see the definition and it clear that I sort of glossed over the `func ()` part and by the time I was it was complete gone. Retracting the comment.